docs/source/dev_guide/includes/parameter_numeric_table.rst

```diff
@@ -0,0 +1,21 @@
+.. ******************************************************************************
+.. * Copyright 2014-2020 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. note::
+
+    This parameter can be an object of any class derived from ``NumericTable``
+    except ``PackedTriangularMatrix``, ``PackedSymmetricMatrix``, and
+    ``CSRNumericTable``.
```

Comment:
Do we need dedicated file for that?

Reply:
these notes appear throughout the doc, it's easier to reference them each time that copy-paste

Reply:
OK